sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -1459,4 +1459,13 @@ public Flux<ByteBuffer> toFluxByteBuffer() {
     public Long getLength() {
         return content.getLength();
     }
+
+    /**
+     * Returns a flag indicating whether the content can always be repeatedly consumed using all accessors including
+     * {@link #toStream()} and {@link #toFluxByteBuffer()}
+     * @return a flag indicating whether the content can always be repeatedly consumed using all accessors.
```

Comment:
nit: We should probably not claim that it'll "always" be available. For e.g. for FileContent, we don't read the file contents into memory eagerly and rely on the file to not be deleted to be replayable.